WordPress/WordPress.xcodeproj/project.pbxproj

```diff
@@ -4414,14 +4432,20 @@
 				ENABLE_USER_SCRIPT_SANDBOXING = YES;
 				GCC_C_LANGUAGE_STANDARD = gnu17;
 				GCC_DYNAMIC_NO_PIC = NO;
+				GCC_PREFIX_HEADER = ../Sources/WordPressData/WordPressData_Prefix.pch;
 				GCC_PREPROCESSOR_DEFINITIONS = (
 					"DEBUG=1",
 					"$(inherited)",
+					"WPDATA=1",
```

Comment:
(nit) seems to be unused

Reply:
Good catch! Thanks!

https://github.com/wordpress-mobile/WordPress-iOS/pull/24517